admin/sql/CreateFunctions.sql

```diff
@@ -1753,7 +1753,7 @@ BEGIN
         SELECT DISTINCT ON (arg.artist, rg.id)
             arg.is_track_artist,
             arg.artist,
-            bool_and(r.status IS NOT NULL AND r.status != 1),
+            bool_and(r.status IS NOT NULL AND r.status != 1 AND r.status != 5),
```

Comment:
Maybe add a short comment here like "Withdrawn releases were once official by definition" just to clarify how it's calculated?